### PR TITLE
Feature/admin login

### PR DESCRIPTION
**Pull request by techstackmedia**
# Creation of Login Page and its Context Providers

- Add a Login Page
- Creation of a Context Provider

---

Screenshot 2023-03-07 at 05-31-29 React App

### CODE REVIEW

File: src/components/Admin/Login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    if (email === '' || password === '') {
+      navigate('/admin/login')
+    } else {
+      navigate('/')
+    }
+  }
+
+  const cardStyles = useBreakpointValue({
+    base: {},
+    md: {
```
Comment (dewale005) [inline src/components/Admin/Login/Login.tsx:69]: Add this to the application theme

File: src/components/Admin/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      boxShadow: 'none',
+    },
+    sm: {
+      height: '100vh',
+      width: '100%',
+      borderRadius: '0',
+    },
+  })
+
+  const add = {
```
Comment (dewale005) [inline src/components/Admin/Login/Login.tsx:82]: Add this to the global style

**Review by dewale005 — CHANGES_REQUESTED** [review]
Make a global css file and add all your style to that file

File: src/components/Admin/Login/Login.tsx (hunk 1 above)
Comment (techstackmedia) [reply src/components/Admin/Login/Login.tsx:69]: The rule only applies to the Login page

File: src/components/Admin/Login/Login.tsx (hunk 2 above)
Comment (techstackmedia) [reply src/components/Admin/Login/Login.tsx:82]: Same as this

**Pull request closed without merging**